README.md

```diff
@@ -273,3 +273,96 @@ the HTML on the page.
 
 This is a non-trivial task and there may be room for improvement in this
 library to make this easier. If you have ideas, please open an issue!
+
+## Knowledge base - usage with third party libraries
+### Select2
+If you're using [Select2](https://select2.org/) that is instantiated on the page load, you may need to wrap it in a Stimulus controller to make it work when new fields dynamically pop up into your HTML DOM.  
+Example: [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames) + [Custom stimulus controller](https://stimulus.hotwired.dev/reference/controllers)
+
+**_select2_controller.js_**
```

Comment:
No need in this as the file name along with the path is specified in the code block itself